backend/src/test/java/ch/puzzle/okr/controller/ObjectiveControllerIT.java

```diff
@@ -341,16 +341,13 @@ void shouldThrowExceptionWhenObjectiveWithIdCantBeFoundWhileDeleting() throws Ex
                 .andExpect(MockMvcResultMatchers.status().isNotFound());
     }
 
-    @DisplayName("Should return is created when an ojective was duplicated")
+    @DisplayName("Should return is created when an objective was duplicated")
     @Test
     void shouldReturnIsCreatedWhenObjectiveWasDuplicated() throws Exception {
-        BDDMockito
-                .given(objectiveAuthorizationService.duplicateEntity(anyLong(), any(), anyList()))
-                .willReturn(objective1);
         BDDMockito.given(keyResultMapper.toDto(any(KeyResultMetric.class), any())).willReturn(keyResultMetricDto);
         BDDMockito.given(keyResultMapper.toDto(any(KeyResultOrdinal.class), any())).willReturn(keyResultOrdinalDto);
         BDDMockito.given(objectiveAuthorizationService.getAuthorizationService()).willReturn(authorizationService);
-        BDDMockito.given(objectiveMapper.toDto(objective1)).willReturn(objective1Dto);
+        BDDMockito.given(objectiveMapper.toDto(any())).willReturn(objective1Dto);
```

Comment:
be careful using any here, could lead to unexpected behavior. Generally it's considered to be best practice to mock as precise as possible.